python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -757,6 +757,56 @@ class FgcmFitCycleConfig(pipeBase.PipelineTaskConfig,
         default=None,
         optional=True,
     )
+    deltaAperFitMinNgoodObs = pexConfig.Field(
+        doc="Minimum number of good observations to use mean delta-aper values in fits.",
+        dtype=int,
+        default=2,
+    )
+    deltaAperFitPerCcdNx = pexConfig.Field(
+        doc="Number of x bins per ccd when computing delta-aper background offsets.",
+        dtype=int,
+        default=10,
+    )
+    deltaAperFitPerCcdNy = pexConfig.Field(
+        doc="Number of y bins per ccd when computing delta-aper background offsets.",
+        dtype=int,
+        default=10,
+    )
+    deltaAperFitSpatialNside = pexConfig.Field(
+        doc="Healpix nside to compute spatial delta-aper background offset maps.",
+        dtype=int,
+        default=64,
+    )
+    deltaAperInnerRadiusArcsec = pexConfig.Field(
+        doc="Inner radius used to compute deltaMagAper (arcseconds).",
+        dtype=float,
+        default=0.0,
+    )
+    deltaAperOuterRadiusArcsec = pexConfig.Field(
+        doc="Outer radius used to compute deltaMagAper (arcseconds).",
+        dtype=float,
+        default=0.0,
```

Comment:
fgcmcal itself doesn't know the radii so I don't want to set to an arbitrary number.  (Though this is actually gettable through a complicated metadata transaction that I did not want to add, though it could be.). The `fgcm` code will take 0s to mean that the output should be "unnormalized" units and I added this to the docstring.  I think that the key is that these should be set in the various pipeline configs for a given dataset.

Reply:
In that case, the `optional` keyword argument should be set to `False` to indicate that a user must ensure that this is provided (I don't know what `default` would do for a non-optional config field though)

Reply:
I'm not sure about that.  You don't need to set this if you aren't doing any of the delta-aper computations.  And the code will run.  But maybe what I need to do is put some code in the validation field that ensures that these are set to positive numbers if any of the `do` options are set.  So I'll do that.

Reply:
Ok, that _is_ tricky then. Checking for positivity within a `validate` method sounds good.